elements/bulbs-quiz/bulbs-quiz-cms.js

```diff
@@ -0,0 +1,17 @@
+import {
+  registerElement,
+  EmbededCMSElement,
+} from 'bulbs-elements/register';
+
+class EmbeddedBulbsQuiz extends EmbededCMSElement {
```

Comment:
So minor but we should eventually fix the misspelling of `EmbededCMSElement` to `EmbeddedCMSElement`